### ISSUE TITLE
Fleetctl no longer supported on AWS CoreOS

### ISSUE THREAD

**Issue by duaneke** [tleyden/open-ocr#100]
Managed to kick-off a cluster on AWS.
But run into problems here:

`./start-fleet.sh`

Everything executed properly, and then I get the following error:

`Starting units ..
./start-fleet.sh: line 13: fleetctl: command not found`

Investigating this I found that fleetctl has been deprecated and is no longer supported on CoreOS.

https://github.com/coreos/fleet

Apparently fleet has been deprecated in favor of Kubernetes.

1. This is a nice project I believe. Notwistanding a few other bugs I found when deploying this, it need to be updated to support Kubernetes on AWS?
2. Any suggestions on how I can still run this using Kubernetes while the `./start-fleet.sh` et al. is updated?

**Comment by tleyden** [comment]
Yes this definitely needs to be updated to support Kubernetes.   If you want to drive this forward, I'll promise to merge in a prompt manner.  Since it's mainly additive it won't need to go through much review.
